lxqt-session/src/sessionapplication.cpp

```diff
@@ -123,8 +123,12 @@ bool SessionApplication::startup()
                 });
 #endif
     }
+    // Avoid showing error with kwin
+    QByteArray envValue = qgetenv("XDG_CURRENT_DESKTOP");
+    QString env = QString::fromLocal8Bit(envValue);
+    bool isKwin = env.contains(QStringLiteral("kwin_wayland"));
 
-    if (lockScreenManager->startup(isX11 ? settings.value(QLatin1String("lock_screen_before_power_actions"), true).toBool() : false
+    if (lockScreenManager->startup(isKwin ? false : settings.value(QLatin1String("lock_screen_before_power_actions"), true).toBool()
```

Comment:
Making an exception for `kwin_wayland` makes the code "dirty" and hard to maintain.

Reply:
If we don't add this exception every time user suspend they will see 
```Failed to run  \"%1\". "
                     "Ensure the specified locker/screensaver is installed and running."``` at wakeup. If you have another idea how to avoid this we can remove it there. But as kwin isn't wlroots we need an exception somewhere.

Reply:
Actually we do a[ similar thing here](https://github.com/lxqt/lxqt-config/pull/995) for `lxqt-config-monitor`: showing the error on wlroots but not on kwin_wayland.

Reply:
Found the command under kwin which works :)
`loginctl lock-session`